CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Documentation for rocPRIM is available at
 
 ### Changed
 
+* Tuned device segmented radix sort for gfx942.
```

Comment:
To make this useful to end users, I would put in some information about how this changed things. The way this reads right now, it sounds more like an addition than a change.

Reply:
I moved it to the additions section and added a note that it improves performance.